cmd/ocm-backplane/cloud/assume.go

```diff
@@ -62,6 +68,10 @@ type namedRoleArn struct {
 }
 
 func runAssume(_ *cobra.Command, args []string) error {
+	if args[0] == "" && assumeArgs.debugFile == "" {
```

Comment:
Seems it is panic

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/openshift/backplane-cli/cmd/ocm-backplane/cloud.runAssume(0x47965e0?, {0x48abc80?, 0x0?, 0x0?})
	/home/bmeng/NVMe/go/src/github.com/openshift/backplane-cli/cmd/ocm-backplane/cloud/assume.go:71 +0xcdb
github.com/spf13/cobra.(*Command).execute(0x47965e0, {0x48abc80, 0x0, 0x0})
	/home/bmeng/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x4798b40)
	/home/bmeng/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/home/bmeng/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
main.Execute()
	/home/bmeng/NVMe/go/src/github.com/openshift/backplane-cli/cmd/ocm-backplane/root.go:56 +0x25
main.main()
	/home/bmeng/NVMe/go/src/github.com/openshift/backplane-cli/cmd/ocm-backplane/main.go:20 +0x17

Reply:
The test I had been running provided both the debug flag and the arg 🤦🏻 , good catch.

Fixed in my latest commit.